src/abilities/Vehemoth.js

```diff
@@ -284,101 +283,49 @@ export default (G) => {
 			},
 		},
 
-		// 	Thirt Ability: Snow Storm
+		/**
+		 * Third Ability: Flake Convertor
+		 *
+		 * Ranged attack on a fatigued and inline enemy unit within 5 range. Deals damage
+		 * equal to the positive Frost mastery difference between Vehemoth and the target,
+		 * who also receives the "Frozen" status.
+		 *
+		 * When upgraded, the "Frozen" status becomes "Cryostasis" which is a special
+		 * "Freeze" that is not broken when receiving damage.
+		 *
+		 * Targeting rules:
+		 * - The target must be an enemy unit.
+		 * - The target must have the "Fatigued" status (no remaining endurance).
+		 * - The target must be inline (forwards or backwards) within 5 range.
+		 * - The path to the target unit cannot be interrupted by any obstacles or units.
+		 *
+		 * Other rules:
+		 * - Attacked unit receives the "Frozen" status, making them skip their next turn.
+		 * - There is no cap to the damage dealt from the positive Frost mastery difference.
+		 * - Attack damage will be 0 if the target has a higher Frost mastery than Vehemoth,
+		 *   but the "Frozen"/"Cryostasis" effect will still be applied.
+		 * - The upgraded "Cryostasis" effect does not break when receiving damage from
+		 *   the Vehemoth or ANY other source.
+		 */
 		{
-			//	Type : Can be "onQuery","onStartPhase","onDamage"
 			trigger: 'onQuery',
 
 			_targetTeam: Team.enemy,
+			_directions: [1, 1, 1, 1, 1, 1],
+			_distance: 5,
```

Comment:
@DreadKnight Is there a place in `units.json` for this kind of data to go?

Reply:
@nandastone We could fit that stuff in `units.json` if we want to, though usually directions wouldn't change and for known common patterns we could probably just use a variable rather (`standard` perhaps).